todo/src/App.js

```diff
@@ -1,29 +1,50 @@
-// import logo from './logo.svg';
 import {useState} from 'react';
-
-import logo from './tek_logo.png';
 import './App.css';
-import Todo from './Components/Todo';
+import TodoList from './Components/TodoList';
+import TodoForm from './Components/Form'
+
+
 
 function App() {
 
-  const [todos, setTodos] = useState(["eat","sleep","code","pray","repeat"]);  //sample todo items.  these will need to be changed in your app.  Just filler todos, although they are quite important!
+    const [todos, setTodos] = useState([]); 
+  
+    const deleteHandler = (id) => {
```

Comment:
If you create 2 equal items by mistake and try to delete one of them, they both get deleted.